### PR TITLE
new project chapter in Userguide/Overview

### PR DESCRIPTION
**Pull request by urskaufmann**
Describes important properties in TWW-project

### CODE REVIEW

**Comment by urskaufmann** [comment]
I can not imagine, why these "interlis" files are in this pull request. I have just changed files in the doc's. ???
What do I have to do now?

**Comment by ponceta** [comment]
> I can not imagine, why these "interlis" files are in this pull request. I have just changed files in the doc's. ??? What do I have to do now?

I'll merge 291 and then this should reflect only your latests commits.

File: docs/en/user-guide/layerexplanations/twwproject.rst [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+Relations are necessary for the Feature Attributes Windows to referende to connected tables (e.g. you can edit connected maintenance_events in Feature_Attribut window in vw_tww_reach).
+Where possible, in TWW are value relations used and not project relations. Value Relations ar used for Value Lists, but also for organisations and for pipe_profiles.
+
+**Value Relations** work perfect also with Multiedit, but is slow, if there are a lot records in the related table or list. And you can not choose a record on the map to connect to it (e.g to define a manhole-catchment area-connection).
+
+**Project relations** allow to select a record on the map and is faster with big datasets. But there is a serious bug when using Multiedit (as of today with QGIS 3.34.4).
+If there is a good advantage to work with Multiedit, it's a good idea to define a style, that has no relation-field in the Feature Attributs window.
+
+
+Variables
```
Comment (ponceta) [inline docs/en/user-guide/layerexplanations/twwproject.rst:31]: Moved to an attention point that can be removed when QGIS behaviour becomes stable again when dealing with Project relations

**Review by ponceta — APPROVED** [review]
In a general matter we will try to rely as much as possible on QGIS official documentation when dealing with QGIS specific widgets and stuff but for now I'm very satisfied of what you propose! (always room for improvement but it's a great step)

Thank you @urskaufmann

**Pull request merged**